CONTRIBUTION.md

```diff
@@ -24,14 +24,14 @@ We welcome contributions of:
       * Unit Tests (JUnit / Java)
       * Acceptance Tests (docker + robot framework)
       * Blockade tests (python + blockade) 
-      * Performance: We have multiple type of load generator / benchmark tools (`ozone freon`, `ozone genesis`). Which can be used to test cluster and report problems.
+      * Performance: We have multiple type of load generator / benchmark tools (`ozone freon`, `ozone genesis`), which can be used to test cluster and report problems.
  * **Bug reports** pointing out broken functionality, docs, or suggestions for improvements are always welcome!
  
 ## Who To Contact
 
 If you have any questions, please don't hesitate to contact
 
- * in **mail**: use hdfs-dev@hadoop.apache.org.
+ * **email**: use hdfs-dev@hadoop.apache.org.
  * in **chat**: You can find the #ozone channel at the ASF slack. Invite link is [here](http://s.apache.org/slack-invite) 
```

Comment:
Remove `in`. Sorry I don't know how I missed it in my earlier review.

Reply:
Thanks, it's also updated.